src/locales/i18n.en.json

```diff
@@ -14,35 +14,35 @@
       "dontHaveAccount": "Don't have an account?",
       "enterEmail": "Enter your email...",
       "haveAccount": "Already have an account?",
-      "iAccept": "By continuing, you agree to Endpass's",
```

Comment:
at this file we have `non-breaking space` with ASCII 160 and `defaul space` with ASCII 32.
not sure what will be better

Reply:
If we need non-breaking space, better will be remove black cat from dark room and replace it with specific symbol explicitly, because there are no opportunity to understand what's kind of space using

Reply:
I have experimented with it. Only `\u00A0` works correctly with vue without `v-html`. But now we have an issue with BabelEdit translator. It removes all special characters

Reply:
I guess, we trying solve problem from wrong side. Every time when you use non-breaking space in layout, you create unflexible, rigid place which will be very hard to customise via CSS. You just moving in HTML responsibility zone of properties like `white-space` and some other, This way makes reusing and styling difficult and dirty, suffers responsibility. Content always first, so we should not restrict content for design.

Instead, we can adapt design for content. We can use CSS for making lines non-breakable, and we can turning on or off this behavior when we really need it. We can build layout which will have enough size for content and will not break line earlier than required; which will stand "content overflow" test.

Reply:
If we will use html approach we will need a lot of splitted strings or html tags in translates. Both are ugly

Reply:
I not suggest html tags in translates, it's ugly - agreed. I just want to say, that we can use CSS instead of non-breakable spaces.

Reply:
How can we split via css `You need accept a document` without breaking after `you` and `a`?

Reply:
This text should be flexible

Reply:
Why break after "you" and "a" appears? Render engine is a pretty smarty, and usually breaks appears if width of containing container less than width of content. When you make content non-breakable (by any way), you break this mechanism, and render engine will try to fit container for content. If you overdo it, non-breakable content may even break too narrow container.

So decision is a pretty simple - you just should take a control over layout. For example, when using flex, there are useful props like `flex-grow` (which allows you to build layouts which can dynamically changes depends of content), or `max-content` width.

If your text breaks like 
```
You 
need 
accept 
a 
document
```
you definitely have too narrow container. In any other cases text will be broke like that
```
You need accept
a document
```
It never be broken like that
```
You
need accept a
document
```

> This text should be flexible

But doesn't non-breaking spaces make it unflexible?

Reply:
But text can be splitted as `you need to accept a` and `document`. It breaks english rules for breaking

Reply:
> It breaks english rules for breaking

This is everywhere, in any site, on reddit, stackoverflow and bbc, in every e-book that you can buy. Because there aren't a way to handle these cases properly, and authors cannot to put `&nbsp` after articles in their posts. Maybe, only in formal business English this is bad tone.

Anyway, we should not remove special chars. Very implicit. Why we move from previous variant with `\u00A0`? Because BabelEdit removes it?

We can control cases like that with accuracy and track these cases (how it appears). This is most simple solution. Also we can replace `&nbsp` before using `vue-l18n` with our proxy over native `$t`, if we really want to clue article with word.

Reply:
BabelEdit has removed. Heather will use it. And it's not good :disappointed: